### PR TITLE
feat : added buttons feature

### PR DESCRIPTION
**Pull request by vivekgudipudi**
image
image
image
image
image

### CODE REVIEW

**Review by MiryalaAkshitha — APPROVED** [review]
good to go!

File: components/buttons/buttons.css [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    font-weight: var(--bold);
+}
+
+
+.link-primary-color{
+    color: var(--primary-color);
+    border-bottom: 1px solid var(--primary-color);
+    background-color: var(--background-color);
+    font-weight: var(--bold);
+    border-radius: 0;
```
Comment (krishanu2911) [inline components/buttons/buttons.css]: Just a small suggestion that here if border-radius: 0 can be removed them remove it will make your code cleaner

**Review by krishanu2911 — APPROVED** [review]
Nice Well Written code and Good use of CSS variables and class names.
Just add a small suggestion Everything is good to go👍🏻

File: components/buttons/buttons.css (hunk 1 above)
Comment (vivekgudipudi) [reply components/buttons/buttons.css]: Oops!! I missed it, will remove that now, Thanks

**Pull request merged**